nautobot/extras/datasources/git.py

```diff
@@ -117,6 +112,19 @@ def get_repo_from_url_to_path_and_from_branch(repository_record):
             from_url = re.sub("//", f"//{quote(user, safe='')}:{quote(token, safe='')}@", from_url)
         else:
             from_url = re.sub("//", f"//{quote(token, safe='')}@", from_url)
+    return from_url, repository_record.remote_url
```

Comment:
I don't quite understand the advantage of using this getter method (`_, remote_url = get_repo_url(repository_record)`) over just accessing `repository_record.remote_url` directly?